src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software':'8.13.7',
+	'confluence':'7.12.2',
+	'stash':'7.12.1',
+}
+
+def get_lts_version(argv):
+	lts_version=''
+	product = argv[0].lower()
+	if product == 'jira':
+		product = 'jira-software'
+	elif product == 'bitbucket' :
+		product = 'stash'
+
+	if product in known_lts_version:
+		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+
+		try:
+			fl = urllib.request.urlopen(url)
+			fdata = fl.read()
+			jsdata = json.loads(fdata[10:len(fdata)-1].decode("utf-8"))
+			enterprise_edition = [x for x in jsdata if x['edition'].lower() == 'enterprise']
+			sortedData = sorted(enterprise_edition, key=lambda k:cversion(k['version']), reverse=True)
+
+			if len(sortedData)>0 :
+				lts_version = sortedData[0]['version']
+			# print (f"Succeed!  LTS Version for {argv[0]} is {lts_version}")
```

Comment:
not sure if this is intended to be commented out? If so we should remove it

Reply:
We can remove these but I left this on purpose to be used for debug if needed in future

Reply:
I think we should remove them

Reply:
Already removed them